common/artifacts/901_wc_artifacts_weapon_twohanded.txt

```diff
@@ -235,7 +235,39 @@ scepter_of_sargeras = {
 
 	text_effect_powerful_magical_artifact = 1
 
-	picture = "GFX_inv_staff_2h_artifactsargeras_d_01"
+	picture = "GFX_scepter_of_sargeras"
+	slot = weapon
+}
+warblade_of_archimonde = {
+	quality = 5
+	flags = { combat_weapon sword fel_artifact }
+
+	active = {
+		is_adult = yes
+
+		OR = {
+			NOT = { has_class_trait_trigger = yes }
+			is_physical_class_trigger = yes
+			is_combined_class_trigger = yes
+			has_character_flag = is_archimonde
```

Comment:
Maybe we just could make Archimonde half-warrior half-warlock? Or not half... anyway, you got my point.

Reply:
well he's definitely a warlock, so i think having an exception for this item is ok

Reply:
If he can wield sword, he's kinda warrior. +1 warrior level won't break anything.

Reply:
well the blade is from rpg lore, and its only mentioned that kazzak uses archies blade after he died

Reply:
So, RPG lore says Archimonde is kinda warrior.
![image](https://user-images.githubusercontent.com/46384992/71547471-c0e2d080-29b9-11ea-88e8-a7e9392331f0.png)

Reply:
Also, I can use the blade, but the tooltip can't say why.
<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46384992/71547539-947b8400-29ba-11ea-83dc-130874880ef1.png)

</details>